README.md

```diff
@@ -12,9 +12,6 @@ and then
 
 `npm i`
 
-This will automatically install package dependencies with `lerna bootstrap`.
-It will also automatically compile the submodules like react-geo, the state and components module via postinstall.
-
```

Comment:
There are some other `lerna` related changes needed in the README, see e.g. chapter `How to add a dependency?`. Please also check the current state of the listed `TODOs`.